### ISSUE TITLE
Auto import inserts import into a comment

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/TypeScript#20293]
_From @yume-chan on November 22, 2017 2:8_

- VSCode Version: Code - Insiders 1.19.0-insider (d294b25f2dd5c96d89bd23f82bd5fd2c0958a4bd, 2017-11-21T06:34:51.660Z)
- OS Version: Windows_NT x64 10.0.16299
- Extensions: Extensions are disabled
---

Steps to Reproduce:

1. Create a new folder
1. Open in Code
2. `npm i @types/node`
3. Create `index.js`
4. Type `/// <reference types="node" />`
5. Type `setTimeout`, select the auto import suggest.

Expected Result:

````JavaScript
/// <reference types="node" />

import { setTimeout } from "timers";

setTimeout
````

Actual Result:

````JavaScript
/// <reference types="node" />import { setTimeout } from "timers";



setTimeout
````

_Copied from original issue: Microsoft/vscode#38914_

**Comment by mjbvz** [comment]
Only seems to happen on windows. Here's the code action we get back:

```
 "codeActions": [
            {
                "description": "Import 'setTimeout' from \"timers\".",
                "changes": [
                    {
                        "fileName": "c:/source/test/y.js",
                        "textChanges": [
                            {
                                "start": {
                                    "line": 1,
                                    "offset": 33
                                },
                                "end": {
                                    "line": 1,
                                    "offset": 33
                                },
                                "newText": "import { setTimeout } from \"timers\";\r\n\r\n"
                            }
                        ]
                    }
                ]
            }
        ],
```

**Comment by mhegazy** [comment]
From https://github.com/Microsoft/TypeScript/issues/20292#issuecomment-347473694

   ```js
   import { read } from "fs";
   
   #!/usr/bin/env node
   ```

**Comment by andy-ms** [comment]
I'm not reproducing this with `typescript@next` on windows in a CRLF file.

**Comment by oliversalzburg** [comment]
The case that was quoted in https://github.com/Microsoft/TypeScript/issues/20293#issuecomment-347608687 appeared on my workstation.

That happened on Windows in a file with LF.

**Comment by weswigham** [comment]
I could swear that @amcasey mentioned noticing this before and having a fix for it; Am I wrong?

**Comment by andy-ms** [comment]
I'm unable to reproduce either error (the original or the shebang one which should have been fixed by #20306) on windows in either an LF or CRLF file.
